cfgov/ask_cfpb/tests/test_blocks.py

```diff
@@ -13,19 +15,28 @@ def setUp(self):
             'type': 'tip',
             'value': self.tip_content
         }
-        self.expected_tip_html = (
-            '<aside class="m-inset m-inset__bordered">'
-            '<h4>Tip</h4>'
-            '<div class="rich-text">Tip content</div>'
-            '</aside>'
-        )
+        if wagtail.VERSION < (2, 10):
+            self.expected_tip_html = (
+                '<aside class="m-inset m-inset__bordered">'
+                '<h4>Tip</h4>'
+                '<div class="rich-text">Tip content</div>'
+                '</aside>'
+            )
+            self.expected_text_html = '<div class="rich-text">text</div>'
+        else:
+            self.expected_tip_html = (
+                '<aside class="m-inset m-inset__bordered">'
+                '	<h4>Tip</h4>'
+                '	Tip content'
+                '</aside>'
+            )
+            self.expected_text_html = 'text'
         self.text_data = {
             'type': 'text',
             'value': {
                 'content': 'text'
             }
         }
-        self.expected_text_html = '<div class="rich-text">text</div>'
```

Comment:
This one in particular seems like it could affect rendering.

Reply:
@willbarton This change to ask_cfpb may impact the way block renders for users. This will require more investigation and browser testing as a result of the HTML `rich-text` changes that wagtail are making. I am very interested to see if the new cypress tests start to fail.

Reply:
The Cypress tests check functionality, not rendering. We don't have visual regression testing yet. Could you check?

You can also run Cypress locally if you want to see what it does.

Reply:
What change to the blocks test are you requesting in this PR? How do you want me to check the rendering?

Reply:
I think the expected HTML should still be `<div class="rich-text">text</div>`. Instead of changing the test, did you try to figure out why the expected output changed?

Reply:
I have not yet been able to figure out why the expected output changed in the expected HTML.

Reply:
I reverted the test back to using `rich-text` class